cmd/app/list.go

```diff
@@ -38,30 +38,31 @@ package app
 
 import (
 	"fmt"
+	"net/http"
+	"sort"
+
+	"github.com/pkg/errors"
+	"gopkg.in/urfave/cli.v2"
+
 	"github.com/VirgilSecurity/virgil-cli/client"
 	"github.com/VirgilSecurity/virgil-cli/models"
 	"github.com/VirgilSecurity/virgil-cli/utils"
-	"github.com/pkg/errors"
-	"gopkg.in/urfave/cli.v2"
-	"net/http"
-	"sort"
 )
 
-func List(vcli *client.VirgilHttpClient) *cli.Command {
+func List(vcli *client.VirgilHTTPClient) *cli.Command {
 	return &cli.Command{
 		Name:    "list",
 		Aliases: []string{"l"},
 		Usage:   "List your apps",
 		Action: func(context *cli.Context) (err error) {
-
 			var apps []*models.Application
 			apps, err = listFunc(vcli)
 
 			if err != nil {
 				return err
 			}
 
-			defaultApp, err := utils.LoadDefaultApp()
+			defaultApp, _ := utils.LoadDefaultApp()
```

Comment:
Why ignore error?

Reply:
the err variable was unused. You can see dafaultApp variable use for setup defaultAppID variable and it is used for mark in application lists an application as default. 
https://github.com/VirgilSecurity/virgil-cli/blob/golangci/cmd/app/list.go#L86